repository/zookeeper/operator/templates/validation.yaml

```diff
@@ -17,8 +17,8 @@ spec:
               value: {{ .Name }}-zookeeper-0.{{ .Name }}-hs:{{ .Params.CLIENT_PORT }},{{ .Name }}-zookeeper-1.{{ .Name }}-hs:{{ .Params.CLIENT_PORT }},{{ .Name }}-zookeeper-2.{{ .Name }}-hs:{{ .Params.CLIENT_PORT }}
           resources:
             requests:
-              memory: "{{ .Params.MEMORY }}"
-              cpu: "{{ .Params.CPUS }}"
+              memory: "64Mi"
+              cpu: "0.1"
```

Comment:
changing to hardcoded values intentional?

Reply:
this an internal validation process, we may want it to go away entirely.

Reply:
oh I just saw this is a merge of `dev -> master` so the change has already been made

Reply:
it's a validation pod, we might introduce later configurable values for this one. But all it has to do is one call to zookeeper and finish its job. It's not a long-running pod and just used in bootstrap.

The issue was that it was using the same resources `{{ .Params.MEMORY }}` and `{{ .Params.CPUS }}` as the zookeeper nodes when it doesn't have the same responsibilities

Reply:
> changing to hardcoded values intentional?

💯 This plan is about validating ZK connection and doesn't have to be parameterized (or use the same amount of cpu/mem as ZK itself). Additionally, using the same amount of cpu/mem doesn't work with our CI plan.

> this an internal validation process, we may want it to go away entirely.

I found it useful. Also, this is pretty much the only showcase _right now_ of why bother with extra plans at all 🤷‍♂